web/marketing/pages/stop-referral-offer.html

```diff
@@ -0,0 +1,36 @@
+<!-- Copyright (C) 2019 Storj Labs, Inc.
+See LICENSE for copying information. -->
+
+{{define "stopReferralOffer"}}
+<div class="modal fade stop-referral-offer-modal mt-5" tabindex="-1" role="dialog" aria-labelledby="confirm" aria-hidden="true">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="row d-flex justify-content-between">
+                <img class="mt-3 ml-5" src="/static/img/warning.png" height="30" width="30"/>
+                <p class="mt-3">
+                        Replace 
+                        <strong>
+                            {{range .ReferralOffers}}{{if eq .Status 2}}{{.Name}}{{end}}{{end}}
```

Comment:
Shouldn't these be separated somehow?

Reply:
In this case the reason it works is because there will only ever be one current offer and one default offer per offer type. Still I don't like the way I did this. I'm going to use your suggestion https://github.com/storj/storj/pull/2225#discussion_r298130490 and create a few boolean evaluator methods I can invoke in my templates to specifically get what I want and avoid messy template logic.